### PR TITLE
Storage node config params descriptions cleanup

### PR DESCRIPTION
**Pull request by mniewrzal**

### CODE REVIEW

File: pkg/piecestore/psserver/config.go [hunk 1]
```diff
@@ -26,10 +26,11 @@ var (
...
 // Config contains everything necessary for a server
 type Config struct {
-	Path                   string        `help:"path to store data in" default:"$CONFDIR"`
-	AllocatedDiskSpace     int64         `help:"total allocated disk space, default(1GB)" default:"1073741824"`
-	AllocatedBandwidth     int64         `help:"total allocated bandwidth, default(100GB)" default:"107374182400"`
-	KBucketRefreshInterval time.Duration `help:"how frequently checker should audit segments" default:"3600s"`
+	Path                         string        `help:"path to store data in" default:"$CONFDIR"`
+	AllocatedDiskSpace           int64         `help:"total allocated disk space in bytes, default(1GB)" default:"1073741824"`
+	AllocatedBandwidth           int64         `help:"total allocated bandwidth in bytes, default(100GB)" default:"107374182400"`
+	KBucketRefreshInterval       time.Duration `help:"how frequently Kademlia bucket should be refreshed with node stats" default:"3600s"`
```
Comment (kaloyan-raev) [inline pkg/piecestore/psserver/config.go]: `3600s` is 1 hour, right? How about changing the default to `1h0m0s` so it's clearer?

Comment (mniewrzal) [reply pkg/piecestore/psserver/config.go]: done

File: pkg/piecestore/psserver/config.go [hunk 2]
```diff
@@ -69,7 +70,7 @@ func (c Config) Run(ctx context.Context, server *provider.Provider) (err error)
 	}()
 
 	//agreementsender
-	agreementsender := agreementsender.New(zap.L(), s.DB, server.Identity(), k)
+	agreementsender := agreementsender.New(zap.L(), s.DB, server.Identity(), k, c.AgreementsenderCheckInterval)
```
[the remark this answers is not in the document]
Comment (mniewrzal) [reply pkg/piecestore/psserver/config.go]: done

File: pkg/piecestore/psserver/config.go [hunk 3]
```diff
@@ -26,10 +26,11 @@ var (
...
 type Config struct {
-	Path                   string        `help:"path to store data in" default:"$CONFDIR"`
-	AllocatedDiskSpace     int64         `help:"total allocated disk space, default(1GB)" default:"1073741824"`
-	AllocatedBandwidth     int64         `help:"total allocated bandwidth, default(100GB)" default:"107374182400"`
-	KBucketRefreshInterval time.Duration `help:"how frequently checker should audit segments" default:"3600s"`
+	Path                         string        `help:"path to store data in" default:"$CONFDIR"`
+	AllocatedDiskSpace           int64         `help:"total allocated disk space in bytes, default(1GB)" default:"1073741824"`
+	AllocatedBandwidth           int64         `help:"total allocated bandwidth in bytes, default(100GB)" default:"107374182400"`
+	KBucketRefreshInterval       time.Duration `help:"how frequently Kademlia bucket should be refreshed with node stats" default:"3600s"`
+	AgreementsenderCheckInterval time.Duration `help:"duration between agreement checks" default:"1h0m0s"`
```
[the remark this answers is not in the document]
Comment (mniewrzal) [reply pkg/piecestore/psserver/config.go]: done

**Pull request merged**